core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/runners/BPMBillingProjectDeleteRunner.scala

```diff
@@ -92,16 +92,13 @@ class BPMBillingProjectDeleteRunner(
   ): Future[JobStatus] =
     Try(workspaceManagerDAO.getDeleteLandingZoneResult(jobId.toString, lzId, ctx)) match {
       case Failure(e: ApiException) =>
-        val (msg, jobStatus) = e.getCode match {
-          case 500 => (s"Landing Zone deletion operation with jobId $jobId failed: ${e.getMessage}", Incomplete)
-          case 404 => (s"Unable to find jobId $jobId in WSM for Landing Zone deletion", Complete)
-          case code =>
-            (
-              s"API call to get Landing Zone deletion operation failed with status code $code: ${e.getMessage}",
-              Incomplete
-            )
+        val msg = s"API call to get Landing Zone deletion operation failed with status code ${e.getCode}: ${e.getMessage}"
+        e.getCode match {
+          case code if code > 400 && code < 500 =>
```

Comment:
Did you intentionally leave 400 out of this?

Reply:
Kind of. I was thinking of 401, 403, and 404 as the important ones to catch (those are the only non-500 exceptions I've seen from any of the async job endpoints, and I didn't see any others in the service code). 

So it wasn't exactly intentional, but it also wasn't exactly a logic error.

I can make it inclusive of 400; it's only the 500 and 404 that I feel strongly about personally, since those are what I've actually encountered with these endpoints (also why I have explicit tests for them).
It does look like it's an off-by-one error like that.